selftests/unit/utils/software_manager.py

```diff
@@ -13,12 +13,26 @@ def apt_supported_distro():
     return distro.detect().name in ["debian", "Ubuntu"]
 
 
+def login_binary_path(distro_name, distro_version):
+    """Retrieve the login binary path based on the distro version""""
+    if distro_name == "Ubuntu":
+        if float(distro_version) >= 24.04:
+            return "/usr/bin/login"
+    if distro_name == "debian":
+        if distro_version == "trixie":
```

Comment:
This ties into my previous point.  It's very probably that `unstable` also has the file at the same place as `testing`, right?  In the future, when those are released, it'd make sense to do a numerical version comparison IMO.